FirebaseInAppMessagingSwift.podspec

```diff
@@ -24,7 +24,7 @@ See more product details at https://firebase.google.com/products/in-app-messagin
   s.prefix_header_file      = false
 
   s.source_files = [
-    'FirebaseInAppMessaging/Swift/Source/**/*.swift',
+    'FirebaseRemoteConfigSwift/Sources/**/*.swift',
   ]
 
   s.test_spec 'unit' do |unit_tests|
```

Comment:
How about changing line 38 to 10.14-beta to minimize the surface area? Also, make a similar change for the other Swift podfiles

Reply:
good idea!

Reply:
Shouldn't it be whatever version we release it in? So, 10.15-beta if this goes out for 10.15.0?

Reply:
Yes.

Reply:
Actually 10.16